lib/mongorito.js

```diff
@@ -753,6 +753,36 @@ Model.count = function (query) {
 };
 
 /**
+ * Get distinct
+ *
+ * @param {String} field for distinct
+ * @param {Object} query - query to filter the results
+ * @see http://mongodb.github.io/node-mongodb-native/2.0/api/Collection.html#distinct
+ * @api public
+ */
+
+Model.distinct = function (field, query) {
+	// collection, model
+	return new Query(this._collection(), this).distinct(field, query);
```

Comment:
I am not sure why indentation is not showing up in diff, but if you view file, you can see that return in one tab indented same as other methods in the file.